Добавил базовый перевод доки (part 2) (#241)
feature-sliced/documentation: README.md

# feature-sliced

> > `WIP:` The current version of the methodology is under development and some details *may change*

Author: Перевел ридмиху, ее бы оставить в норм состоянии - поэтому на нее можете прям набрасываться

![feature-sliced-banner](/website/static/img/banner.jpg)

<!-- ⚡ Add primary information & features about your repository -->
A methodology for designing *frontend projects*, aimed [**at dividing the application according to business logic and areas of responsibility.**][refs-splitting]

> - Provides [**explicitness, controllability and adaptability**][refs-arch-req] architecture

Reviewer: Кажется, либо так
```suggestion
- Provides [**explicit, controllable and adaptable**][refs-arch-req] architecture
```
либо
```suggestion
- Provides [**explicitness, controllability and adaptability**][refs-arch-req] of architecture
```

- Based on [**experience-checked**][refs-motivation-why] design practices and concepts
    > `SOLID`, `GRASP`, `DDD`, `Separation of Concerns`, `Vertical Slices`, `Public API`, `Isolation`
- Suggests dividing the project according to [**business units**][ext-ubiq-lang]

> **Note:** The methodology is not tied to a specific stack and is applicable *to any frontend projects* in general.
>
> But the current version gives examples and is developed on the basis of a bunch of `JavaScript` + `React`

## Motivation

Usually, the approaches to building the frontend architecture from project to project are [re-invented from scratch][refs-motivation], thereby adding ["project knowledge"][refs-knowledge]

> Despite the fact that the specifics of frontend projects do not differ so much

At the same time, incorrectly made decisions often lead [to problems of scalability of the project and the team][refs-arch-problems].

> And therefore, instead of inventing and documenting it every time, I want to **summarize the experience and form a working, proven and documented methodology** for designing the frontend architecture.

Reviewer: Мб как-то так?
```suggestion
And therefore, instead of inventing and documenting it every time, it is better to **summarize the experience and form a working, proven and documented methodology** for designing the frontend architecture.
```
